### PR TITLE
Python code linters

### PR DESCRIPTION
**Pull request by n0npax**
Python code in minukube is not following pep8 standard.

1) Run automatic python formater on existing python scripts.
2) Add python linters to CI
3) Fix linting in `.travis.yaml`

All python files were modified by `black ./hack`. No logic change happened.

Anyway, are you interested in refactor for python scripts in minikube?

I believe stuff like:
```python
-- 42 def get_major_and_minor():                                                                                              
   43     major = ""                                                                                                          
   44     minor = ""                                                                                                          
   45     version = get_from_godep("Comment")                                                                                 
   46     # [kubernetes/hack/lib/version.sh]:                                                                                 
   47     # Try to match the "git describe" output to a regex to try to extract                                               
   48     # the "major" and "minor" versions and whether this is the exact tagged                                             
   49     # version or whether the tree is between two tagged versions.                                                       
-- 50     m = re.match("^v([0-9]+)\.([0-9]+)(\.[0-9]+)?([-].*)?$", version)                                                   
   51     if m:                                                                                                               
   52         major = m.group(1)                                                                                              
   53         minor = m.group(2)                                                                                              
   54         if m.group(4):                                                                                                  
   55             minor += "+"                                                                                                
   56     return ("gitMajor=%s" % major, "gitMinor=%s" % minor)                     
```
can be optimalized with liblaries like `semver`

### CODE REVIEW

**Comment by afbjorklund** [comment]
> are you interested in

Not really, see #5461

File: .travis.yml [hunk 1]
```diff
@@ -8,15 +9,24 @@ env:
 matrix:
   include:
     - language: python
-      name: Check Boilerplate 
+      name: Code lint - python
```
Comment (medyagh) [inline .travis.yml]: we don't really do minikube code linting, all we really do is checking for license boilerplates. we could rename it to "boilerplate lint".

**Comment by medyagh** [comment]
Ideally I like to see all python codes gone in this repo (check afbjorklund referenced), so we don't need to invest much in the python code but meanwhile it is still good improve the current code which hasn't been touched in years.

**Comment by n0npax** [comment]
seems fair. I forgot to searchfor issues like #5461 . 
Thanks Guys,
Closing PR.

**Pull request closed without merging**